shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -120,6 +122,8 @@ - (void)setUpPlatformViewChannel;
  */
 - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result;
 
+- (void)updateSemantics:(const FlutterSemanticsUpdate*)update forViewId:(uint64_t)viewId;
```

Comment:
I'd rename the parameter `forViewId` to just `view` since we're updating the semantics of the `view` itself, and not the `viewId`

Reply:
`forView` works as well.

Reply:
This method is no longer present in the latest revision. Now `FlutterViewController` has a `updateSemantics:` with no view ID or view parameters.